08_multibundle_requirejs/test/app/collections/repos_bundle/repos.test.js

```diff
@@ -0,0 +1,17 @@
+var should = require("should");
+var Repos = require('../../../../app/collections/repos_bundle/repos');
```

Comment:
I think you can use `global-define` with tests too,
although probably not necessary.